xbmc/music/MusicDatabase.cpp

```diff
@@ -309,7 +311,7 @@ void CMusicDatabase::CreateViews()
               "        album.iUserrating, "
               "        album.iVotes, "
               "        bCompilation, "
-              "        (SELECT MIN(song.iTimesPlayed) FROM song WHERE song.idAlbum = album.idAlbum) AS iTimesPlayed, "
+              "        (SELECT AVG(song.iTimesPlayed) FROM song WHERE song.idAlbum = album.idAlbum) AS iTimesPlayed, "
```

Comment:
Yes, it snucked!
OTOH I do prefer AVG to MIN, and it seems a shame to have a DB version step for a single view change (no updating just need the views recreated as they are with a new db file).

Dare I leave it?